pacehrh/R/pace_stochasticity_config.R

```diff
@@ -30,14 +30,49 @@ loadStochasticParameters <- function(sheetName = .defaultStochasticParametersShe
   return(stochData)
 }
 
+#' Load Change Rate Limits
+#'
+#' @param changeRateLimitsSheetName Sheet name from the model input Excel file
+#'
+#' @return Dataframe of change rate limits
+#'
+#' @noRd
+loadChangeRateLimits <- function(changeRateLimitsSheetName = .defaultChangeRateLimitsSheet) {
+  traceMessage(paste0("Loading change rate limits sheet ", changeRateLimitsSheetName))
+
+  limitData <- tryCatch({
+    readxl::read_xlsx(GPE$inputExcelFile, sheet = changeRateLimitsSheetName)
+  },
+  error = function(e){
+    return(NULL)
+  },
+  finally = {
+  })
+
+  if (is.null(limitData)){
+    warning("Could not read change rate limits sheet")
+    return(NULL)
+  }
+
+  limitData <-
```

Comment:
@celiot-IDM  based on my understanding if the ChangeRateLimits are not set (NAs) it should be consider "no capping"
however, this check seems to return NULL in such scenario and later on result in failure

```
 Uninitialized variables: BVE$changeRateLimits 
In RunExperiments(scenarioName = "BasicServices", trials = 2, debug = TRUE) :
  Critical failure. RunExperiments() ended.
```

Reply:
fixed